### ISSUE TITLE
Confusing DefPageSize

### ISSUE THREAD

**Issue by gromakovsky** [serokell/servant-util#17]
README contains the following snippet:
```
import Servant.Util (PaginationParams, PaginationSpec)
import Servant.Util.Dummy (paginate)

type GetBooks
    :> PaginationParams (DefPageSize 10)
    =  Get '[JSON] [Book]

getBooks :: _ => PaginationSpec -> m [Books]
getBooks pagination = paginate pagination <$> getAllBooks
```

but apparently `PaginationParams` doesn't have any type parameter:
![image](https://user-images.githubusercontent.com/2556409/116217059-f6383380-a751-11eb-8b31-4938cda2cd4a.png)

also it says:
> default value of `limit` is defined in `settings` argument.

What is `settings` argument?

**Comment by Martoon-00** [comment]
Oh meh, that type-level argument was removed at some point for the sake of Disciplina, and the doc got outdated.

Currently, there is no default value, if `limit` is not specified then it is just not applied and all the entries after `offset` (if specified) are returned.

What is your use case? Probably I could bring that back with proper support.

**Comment by gromakovsky** [comment]
Currently I am just studying sorting and pagination capabilities in `servant-util` and found a mistake in documentation. I was not sure hot to fix it (I don't know the story begind `DefPageSize`), but also was not lazy to create this issue :not-bad:

**Comment by gromakovsky** [comment]
One more problem: there is `fieldSort_` in `servant-util-beam-pg`'s README, but no `fieldSort_` in code.

**Comment by Martoon-00** [comment]
Both should be resolved in #25.
